osu.Game/Localisation/ToolbarStrings.cs

```diff
@@ -19,6 +19,22 @@ public static class ToolbarStrings
         /// </summary>
         public static LocalisableString Connecting => new TranslatableString(getKey(@"connecting"), @"Connecting...");
 
+        /// <summary>
+        /// "home"
+        /// </summary>
+        public static LocalisableString HomeHeaderTitle => new TranslatableString(getKey(@"header_title"), @"home");
+
+        /// <summary>
+        /// "return to the main menu"
+        /// </summary>
+        public static LocalisableString HomeHeaderDescription => new TranslatableString(getKey(@"header_description"), @"return to the main menu");
+
+        /// <summary>
+        /// "play some"
+        /// </summary>
+
+        public static LocalisableString RulesetHeaderDescription => new TranslatableString(getKey(@"header_description"), @"play some");
```

Comment:
This needs to take an argument, as depending on language the ruleset's name could move around in the sentence.

If you're using rider or visual studio, you should be able to use the bundled tool to create this:

![JetBrains Rider-EAP 2022-10-12 at 04 54 29](https://user-images.githubusercontent.com/191335/195253486-b0ab6666-d60a-4a46-a0c0-13d6059e660d.gif)

to create something like:

```csharp
        /// <summary>
        /// "play some {0}"
        /// </summary>
        public static LocalisableString PlaySome(string arg0) => new TranslatableString(getKey(@"play_some"), @"play some {0}", arg0);
```